content/en/docs/Core concepts/sharding.md

```diff
@@ -1,23 +1,81 @@
 ---
 title: "Sharding"
-date: 2020-11-02
-weight: 500
+date: 2020-11-03
+weight: 400
+description: >
+  Output sharding
 ---
 
-# TBD - these are just notes
+### Overview
 
-The `[output.sharding]` configuration in the TOML file tells which field of the records must be
-used to calculate the sharding.
+Baker supports partitioning the records it processes into smaller subsets each
+of which is forwarded to an output shard: _divide and rule_.
```

Comment:
divide et impera?

Reply:
I changed to divide and conquer. I've never seen this written in latin.